packages/create-sitecore-jss/src/initializers/nextjs-sxa/index.ts

```diff
@@ -29,6 +33,25 @@ export default class NextjsSxaInitializer implements Initializer {
 
     await transform(templatePath, mergedArgs);
 
+    let addThemeProject: string[] = [];
+
+    // prompt to select which theme projects to include
+    if (args.templates.includes('nextjs-sxa')) {
+      const addThemeAnswer = await inquirer.prompt({
```

Comment:
Can probably remove "// TODO: prompts for SXA" comments above. And should this move up there (answers and be merged to `mergedArgs`)?

Reply:
Same (all good, as discussed)